peps/pep-0756.rst

```diff
@@ -3,11 +3,12 @@ Title: Add PyUnicode_Export() and PyUnicode_Import() C functions
 Author: Victor Stinner <vstinner@python.org>
 PEP-Delegate: C API Working Group
 Discussions-To: https://discuss.python.org/t/63891
-Status: Draft
+Status: Rejected
```

Comment:
Was there a vote to reject here, or is "Withdrawn" more appropriate?

Reply:
I rejected my own PEP, it's not a vote. Should I use "Withdrawn" status instead?

Reply:
Ok, done.